azext_iot/smoke_tests/commands_test.ps1

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# Setting this environment variable enables the az command failures to be thrown and not swallowed
+
+$ErrorActionPreference = "Stop"
+
+$resource_group_name = "cli-int-test-rg"
+$iothub_name = "smoketest-hub-$(New-Guid)"
+$hub_module_id = "smoke-test-module"
+$hub_config_name = "smoke-test-config"
+$hub_config_content = "`"{'moduleContent': {'properties.desired.chillerWaterSettings': {'temperature': 38, 'pressure': 78}}}`""
+
+$device_id = "smoke-test-device"
+$desired_twin_properties = "`"{'conditions':{'temperature':{'warning':70, 'critical':100}}}`""
+
+$edge_deployment_name = "smoke-test-deployment"
+$edge_deployment_content = "'azext_iot\smoke_tests\edge_deployment_content.json'"
+$edge_deployment_metrics = "`"{'queries':{'mymetric':'SELECT deviceId from devices where properties.reported.lastDesiredStatus.code = 200'}}`"" 
+$edge_deployment_condition = "`"tags.environment='dev'`""
+$edge_module_content = "'azext_iot\smoke_tests\edge_module_content.json'"
```

Comment:
Is there a functional difference between the edge module content created for the smoke tests vs the existing content used in tests?

Reply:
There is no functional difference, I could have re-used but didn't want to create a dependency on the existing test content json ( say some test updates are made later, this will be like a hidden dependency). Hence, I felt this is a better design.

Reply:
ok